contracts/digg-core/UFragments.sol

```diff
@@ -302,4 +304,17 @@ contract UFragments is ERC20Detailed, Ownable {
         emit Approval(msg.sender, spender, _allowedFragments[msg.sender][spender]);
         return true;
     }
+
+    /**
+     * @notice Mints the reimbursement for remDIGG one time, directly to dev multisig
```

Comment:
I would add a link to any reference of this issue to the comments to give more insights to new users.